### PR TITLE
Load three youtube videos to the onelive homepage

### LINKED ISSUE #642: Load three youtube videos to the onelive homepage
**Is your feature request related to a problem? Please describe.**
- We need to show three youtube videos on the onelive homepage.

**Describe the solution you'd like**
- Load three youtube videos to the onelive homepage using the youtube API.

### Use `one-live` branch

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #642 

## Goals

- Load three youtube videos to the onelive homepage

## Approach

- Load three youtube videos to the onelive `index.html`

### Screenshots

Screenshot from 2020-04-05 14-39-10
  
### Preview Link

https://pr-643-sef-site.surge.sh/onelive/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Piumal1999** [comment]
Is there a way to get rid of long descriptions?

**Comment by anjula-sack** [comment]
> Is there a way to get rid of long descriptions?

It just for now. I will raise an issue for this.

File: src/main/webapp/onelive/index.html [hunk 1]
```diff
@@ -45,43 +45,22 @@ <h1 class="mt-5 text-onelive" style="font-size: 3.3rem; line-height: 1.5;"><span
...
-                    </a>
-                </div>
-                <div class="col-md-4">
-                    <a href="https://p-upload.facebook.com/sustainableeducationfoundation/videos/506389346839762/" target="_blank">
-                        <img src="images/banner-1.png" class="img-fluid shadow rounded mb-md-4">
-                        <h5 class="text-uppercase">Future-proof - what to learn for the future and how?</h5>
-                        <p>Sohan Darmaraja</p>
-                    </a>
-                </div>
+            <div class="row" id="youtube-video">
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html]: What about renaming this as `youtube-videos`?

File: src/main/webapp/onelive/index.html [hunk 2]
```diff
@@ -45,43 +45,22 @@ <h1 class="mt-5 text-onelive" style="font-size: 3.3rem; line-height: 1.5;"><span
...
+                <!--youtube video goes here-->
             </div>
             <div class="row justify-content-center mb-md-5">
                 <div class="col-9 text-center">
-                    <a href="archive/" class="btn btn-onelive text-uppercase">See More videos</a>
+                    <a href="archive/" class="btn btn-onelive text-uppercase">See more past onelives</a>
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html]: I think just `See more` is sufficient. What do you think?

File: src/main/webapp/onelive/index.html [hunk 3]
```diff
@@ -45,43 +45,22 @@ <h1 class="mt-5 text-onelive" style="font-size: 3.3rem; line-height: 1.5;"><span
...
                 </div>
             </div>
         </div>
-        <img src="images/Asset%205.svg">
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html:84]: Are we going to remove this?

**Review by jayasanka-sack — APPROVED** [review]
Outstanding work!
Thanks @anjula-sack !

**Pull request merged**